### ISSUE TITLE
Can't send "Press" to Bot

### ISSUE THREAD

**Issue by tojannaon** [OpenWonderLabs/python-host#40]
### Current Situation

Although the same Bot can be controlled successfully via the app no response occurs when using sudo python switchbot.py. Note that the device is discovered and connects just fine. It's just that nothing happens on the Bot (no Press).

### Logs

```markdown
pi@raspberrypi:/python-host $ sudo python switchbot.py
Usage: "sudo python switchbot.py [mac dev_type cmd]" or "sudo python switchbot.py"
Scanning...
Scan timeout.
(' 0', [u'd9:31:3b:81:03:56', 'Bot', 'Press'])
Input the device number to control:0
[u'd9:31:3b:81:03:56', 'Bot', 'Press']
Preparing to connect.
Connection successful.
Complete
```

### Configuration

```json
N/A
```

### Environment

- OS: Raspbian
- Software: Jessie
- Node: v10.19.0
- npm: 6.13.4

**Comment by dcarrion87** [comment]
@tojannaon @reestr 

Can you guys can any commands to send including on/off? I'm on version 5.0 and I just get invalid handle errors from gatt.

**Comment by dcarrion87** [comment]
@reestr try this project instead: https://github.com/Danielhiversen/pySwitchbot

I was only able to get it working through that project.

E.g.:

```
from switchbot import Switchbot
from bleak import BleakScanner
import asyncio

async def main():
  ble_device = await BleakScanner.find_device_by_address("MAC ADDRESS", timeout=20)
  device = Switchbot(ble_device)
  await device.press()

if __name__ == "__main__":
  asyncio.run(main())
```

You can also set change mode with:

```
await device.set_switch_mode()
await device.set_long_press(7)
```

**Comment by TForest-UwU** [comment]
Hello,

i have the same issue as @tojannaon, however i the software @dcarrion87 provided gives me this error

> Traceback (most recent call last):
>   File "<string>", line 1, in <module>
> ModuleNotFoundError: No module named 'switchbot'

I have cloned the git through git clone however i cant seem to find a solution to both these problems.

I am using switchbot version 6.3 and a raspberry pi 3 on the raspbian OS

**Comment by dcarrion87** [comment]
@TForest-UwU have you:

```
pip install PySwitchbot
```

Also you'll need these pip packages too:

```
async_timeout>=4.0.1
bleak>=0.17.0
bleak-retry-connector>=2.9.0
cryptography>=38.0.3
boto3>=1.20.24
requests>=2.28.1
```

**Comment by TForest-UwU** [comment]
Yes i retried it and it seemed i also had a outdated pip installation, however the program fails to build cryptography with this error
`  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for cryptography
Failed to build cryptography
ERROR: Could not build wheels for cryptography, which is required to install pyproject.toml-based projects
`
the rest of the packages seem to all have installed correctly

**Comment by dcarrion87** [comment]
Create a [virtual environment](https://docs.python.org/3/library/venv.html) first and work in there so you don't mess with your system wide pip packages.

```
python -m venv venv
. venv/bin/activate
```

Then do your pip things.

You might need to do some googling of that error to make sure you have build-essentials, upgrade pip version, downgrade setuptools if necessary etc...

**Comment by tojannaon** [comment]
@reestr Sorry for not chiming in earlier. I ended up having to bail as the Switchbot folks felt that the Bluetooth LE dongle that I was using (expensive and state-of-the-art) was somehow not compatible (even though it works for other applications). I didn't have another to try and wasn't about to purchase one in case there actually were other issues causing the problem.

**Comment by TForest-UwU** [comment]
If anyone is still having issues check out https://github.com/TForest-UwU/Switchy

I found out that atleast for me this api does not send the command properly, but it does connect nicely.
I then found another api that does the exact opposite, so i combined then into a working program

**Comment by reestr** [comment]
@TForest-UwU - thank you for following up with this, despite reporting an error after each press of the switch, it works as expected! :-)
